package.json

```diff
@@ -29,7 +29,10 @@
     "ember-cli-babel": "^7.1.3",
     "ember-cli-htmlbars": "^3.0.1",
     "ember-cli-sass": "^8.0.1",
-    "uniplaces-uniq": "^2.5.3"
+    "ember-progress-bar": "^1.0.0",
```

Comment:
This is only necessary if you use the `uni-progress-bar` which many projects don't. Therefore, this should not be here

Reply:
That is true but also applies to other dependencies. I think this should be addressed by https://github.com/uniplaces/ember-cli-uniq/issues/279 instead, especially given your own comment at https://github.com/uniplaces/ember-cli-uniq/issues/208.

Reply:
After taking another look, it seems we only use it on `uni-photo-thumbnail`, a component too domain specific that arguably shouldn't even be on ember-cli-uniq anyway. I'll remove the dependency and add a note to remove this in the future.